internal/action/generate.go

```diff
@@ -219,7 +219,10 @@ func (s *Action) generatePassword(ctx context.Context, c *cli.Context, length, n
 	case "xkcd":
 		return s.generatePasswordXKCD(ctx, c, length)
 	case "memorable":
-		return pwgen.GenerateMemorablePassword(pwlen, symbols), nil
+		if c.Bool("strict") {
+			return pwgen.GenerateMemorablePassword(pwlen, symbols, true), nil
+		}
+		return pwgen.GenerateMemorablePassword(pwlen, symbols, false), nil
```

Comment:
@dominikschulz actually we're using `WriteString` in there which can return an error... Why aren't we returning errors in the Generate*Password functions and instead always return `nil` here??

Reply:
It's signature has an error argument, yes.
But at least with the current implementation it will never return an error: https://cs.opensource.google/go/go/+/refs/tags/go1.17.1:src/strings/builder.go;l=125;drc=43652dc46f770253b3603f47165b1568b439b0b5